lte/gateway/deploy/roles/magma/tasks/main.yml

```diff
@@ -61,7 +61,9 @@
     dest: /usr/local/bin/{{ item }}
     force: yes
     remote_src: yes
+    mode: 0755
```

Comment:
is it possible to define a permission mode more restrictive for these scripts?

Reply:
Removed in latest version.